src/Classes/Container.ts

```diff
@@ -181,6 +206,18 @@ export class Container {
         //    Container.cancelAnimation(this.checkAnimationFrame);
         //}
 
+        /* restart */
+        this.stop();
+        await this.start();
+    }
+
+    public stop(): void {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Declaration of public instance method not allowed after declaration of private instance method. Instead, this should come after public static methods.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5179066)